src/BuildScriptGenerator/DefaultBuildScriptGenerator.cs

```diff
@@ -313,6 +315,41 @@ private void LogScriptIfGiven(string type, string scriptPath)
             buildProperties[ManifestFilePropertyKeys.CompressDestinationDir] =
                 _cliOptions.CompressDestinationDir.ToString().ToLower();
 
+            // Override the prebuild and postbuild commands if BuildConfigurationFile exists
+            if (context.SourceRepo.FileExists("app.yaml"))
+            {
```

Comment:
@tulikac may you take a look at the failing tests:
https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=5715683&view=logs&j=e1391e23-97cc-5b71-5818-643679e655ef&t=10188d5a-2cf9-5c06-9eb2-4b00602c03d8&l=19

These changes are failing tests for `DefaultBuildScriptGenerator`